core/platform/email/gae_email_services_test.py

```diff
@@ -43,3 +49,132 @@ def test_sending_email(self):
         messages = self.mail_stub.get_sent_messages(
             to=feconf.ADMIN_EMAIL_ADDRESS)
         self.assertEqual(1, len(messages))
+
+    def test_that_email_bcc_is_sent_when_bcc_admin_is_true(self):
+        with self.swap(feconf, 'CAN_SEND_EMAILS', True):
+            gae_email_services.send_mail(
+                feconf.SYSTEM_EMAIL_ADDRESS, feconf.ADMIN_EMAIL_ADDRESS,
+                'subject', 'body', 'html', bcc_admin=True)
+        messages = self.mail_stub.get_sent_messages(
+            to=feconf.ADMIN_EMAIL_ADDRESS)
+        self.assertEqual(1, len(messages))
+
+    def test_that_email_reply_to_email_is_sent_when_reply_to_id_is_true(self):
```

Comment:
I am not sure what this is testing. The assertion in the test does not seem to be checking anything to do with the reply-to id.

Reply:
Recitified